### PR TITLE
Add GH Action example

### LINKED ISSUE #23: Add Github Actions support / example
Since usage with other CI tools is documented, adding GH actions support makes sense.

### PR DESCRIPTION
**Pull request by Korbeil**
Add a GH Action example on how to use gitsplit !

Fixes #23 
/cc @odolbeau

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,3 +175,37 @@ split:
     - git fetch --prune --unshallow || git fetch --prune
     - gitsplit --ref "${CI_COMMIT_REF_NAME}"
 ```
+
+# Sample with Github Actions
+
+I recommend to not use [the checkout action](https://github.com/actions/checkout) because it will checkout only the last
```
Comment (jderusse) [inline README.md:181]: What's about https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches

```yaml
- uses: actions/checkout@v2
  with:
    fetch-depth: 0
```

**Review by odolbeau — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -175,3 +175,37 @@ split:
...
+commit for performance purpose. In this example we clone the whole repository. Indeed, if you need to split only the
+last changes and you don't need to split tags, you can use it.
+
+Also we do not use the GH Action auto-generated token because it doesn't have the required permissions for gitsplit to
```
Comment (jderusse) [inline README.md:185]: I'm confused, isn't `secrets.GH_TOKEN` the default token?
Would be great to provide a link to the personal page that to generate a new token.
ie. https://github.com/settings/tokens/new?scopes=repo&description=git-split

**Review by jderusse — COMMENTED** [review]
Hey, 

Thank for this contribution. I have few questions :)

File: README.md (hunk 2 above)
Comment (Korbeil) [reply README.md:185]: It's the same name, but I overwrite it, probably confusing, I'll change the secret name :+1: and I'll add a link to generate the token.

File: README.md (hunk 1 above)
Comment (Korbeil) [reply README.md:181]: I did try a lot of combinations when I tried to use checkout action and never had something that had what I needed, if you prefer I can update the README with this syntax but I can't say if it's working or not.

**Comment by jderusse** [comment]
Thank you @Korbeil for this documentation !

**Pull request merged**